Remove WebGLBlockList

achristensen07 opened the pull request:
#### 7ed314827251cfe67743f3c76d52b2b073bead5d
<pre>
Remove WebGLBlockList
<a href="https://bugs.webkit.org/show_bug.cgi?id=259123">https://bugs.webkit.org/show_bug.cgi?id=259123</a>
rdar://112102082

Reviewed by Myles C. Maxfield.

It is unused.

* Source/WebCore/SourcesCocoa.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/platform/mac/BlocklistUpdater.h:
(WebCore::BlocklistUpdater::pluginBlocklist):
(WebCore::BlocklistUpdater::webGLBlocklist): Deleted.
* Source/WebCore/platform/mac/BlocklistUpdater.mm:
(WebCore::BlocklistUpdater::reloadIfNecessary):
* Source/WebCore/platform/mac/WebGLBlocklist.h: Removed.
* Source/WebCore/platform/mac/WebGLBlocklist.mm: Removed.
* Source/WebKit/UIProcess/API/C/mac/WKContextPrivateMac.h:
* Source/WebKit/UIProcess/API/C/mac/WKContextPrivateMac.mm:
(WKContextShouldBlockWebGL): Deleted.
(WKContextShouldSuggestBlockWebGL): Deleted.
* Source/WebKitLegacy/mac/WebCoreSupport/WebFrameLoaderClient.mm:

Canonical link: <a href="https://commits.webkit.org/265970@main">https://commits.webkit.org/265970@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/8704db2c0beed2f0e6e3c713b280a1973433a322

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/12447 "Passed style check") | [  ~~🛠 ios~~](https://ews-build.webkit.org/#/builders/26/builds/12778 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/13091 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/14188 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/11956 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/12506 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/15206 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/12796 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/14643 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/12611 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/15/builds/13354 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/10522 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/14612 "Built successfully") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/39/builds/10636 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/11266 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/18373 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/11720 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 mac-wk2~~](https://ews-build.webkit.org/#/builders/36/builds/11431 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/14630 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/7/builds/11920 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/9846 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [  ~~🛠 tv-sim~~](https://ews-build.webkit.org/#/builders/30/builds/11157 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| | [  ~~🛠 watch~~](https://ews-build.webkit.org/#/builders/4/builds/15487 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/1387 "Built successfully and passed tests") | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/31/builds/11775 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

litherum reviewed (approved):
mechanical dead code removal

achristensen07 committed:
Remove WebGLBlockList
https://bugs.webkit.org/show_bug.cgi?id=259123
rdar://112102082

Reviewed by Myles C. Maxfield.

It is unused.

* Source/WebCore/SourcesCocoa.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/platform/mac/BlocklistUpdater.h:
(WebCore::BlocklistUpdater::pluginBlocklist):
(WebCore::BlocklistUpdater::webGLBlocklist): Deleted.
* Source/WebCore/platform/mac/BlocklistUpdater.mm:
(WebCore::BlocklistUpdater::reloadIfNecessary):
* Source/WebCore/platform/mac/WebGLBlocklist.h: Removed.
* Source/WebCore/platform/mac/WebGLBlocklist.mm: Removed.
* Source/WebKit/UIProcess/API/C/mac/WKContextPrivateMac.h:
* Source/WebKit/UIProcess/API/C/mac/WKContextPrivateMac.mm:
(WKContextShouldBlockWebGL): Deleted.
(WKContextShouldSuggestBlockWebGL): Deleted.
* Source/WebKitLegacy/mac/WebCoreSupport/WebFrameLoaderClient.mm:

Canonical link: https://commits.webkit.org/265970@main
